test/functional/cmdLineTests/ignoreUnrecognizedVMOptions/ignoreunrecognizedxxoptions.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+
+<!--
+  Copyright (c) 2019, 2019 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
```

Comment:
Will also need to test `-XX:+IgnoreUnrecognizedOptions`

Reply:
@tajila did you mean `-XXvm:ignoreUnrecognized`. I couldn't find the option you gave me

Reply:
I meant `-XX:+IgnoreUnrecognizedVMOptions` and `-XX:-IgnoreUnrecognizedVMOptions`

Reply:
There are tests that exist for that seperately in `/test/functional/cmdLineTests/ignoreUnrecognizedVMOptions/ignoreunrecognizedvmoptions.xml` If you're asking to create tests to include both options, then which file should I add them to?

I'll assume that I add them in the suite that test for xx options

Reply:
Ah okay, I wasn't aware that those tests existed, you can ignore this comment

Reply:
Gotcha, I'll delete the tests that I have wrote for this function